subsys/bluetooth/controller/Kconfig

```diff
@@ -239,14 +312,16 @@ if BT_CONN
 
 config BT_CTLR_LE_ENC
 	bool "LE Encryption"
-	depends on !BT_CTLR_DATA_LENGTH_CLEAR && !BT_CTLR_PHY_2M_NRF
+	depends on !BT_CTLR_DATA_LENGTH_CLEAR && !BT_CTLR_PHY_2M_NRF && \
```

Comment:
Is it possible to move the PHY_2M_NRF to where LLL_VENDOR_NORDIC selects ENC_SUPPORT?

`select BT_CTRL_LE_ENC_SUPPORT if BT_CTLR_PHY_2M_NRF`?

Reply:
Yeah, I think that's doable. I will include this in the next push

Reply:
@joerchan , there's something that seems amiss here:  subsys/bluetooth/controller/Kconfig on master contains the following:

`config BT_CTLR_LE_ENC`
`	depends on !BT_CTLR_DATA_LENGTH_CLEAR && !BT_CTLR_PHY_2M_NRF`

The proposed change in this PR is:

`config BT_CTLR_LE_ENC `
`	depends on !BT_CTLR_DATA_LENGTH_CLEAR && BT_CTLR_LE_ENC_SUPPORT`

So I guess that your suggestion above 

`select BT_CTRL_LE_ENC_SUPPORT if BT_CTLR_PHY_2M_NRF` 

should actually be 

`select BT_CTRL_LE_ENC_SUPPORT if !BT_CTLR_PHY_2M_NRF`

Or alternatively, rewrite BT_CTRL_LE_ENC to 

`config BT_CTLR_LE_ENC`
`	depends on !BT_CTLR_DATA_LENGTH_CLEAR && !BT_CTLR_LE_ENC_SUPPORT`

The reason that I've circled back to this is that the BabbleSim tests seem broken and by bisecting the commits, I've reached the one that modifies the Kconfig for controller. Comparing the generated .config on master with the one from this PR singled out the BT_CTRL_LE_ENC config options.

Please let me know if this makes sense.